packages/util/obojobo-lib-utils/bin/obojobo-migrate

```diff
@@ -18,19 +18,23 @@ const fs = require('fs-extra')
 const os = require('os')
 const { execSync } = require('child_process')
 const dbMigratePath = require.resolve('db-migrate/bin/db-migrate')
-const configPath = require.resolve('obojobo-express/server/config/db.json')
+const config = require('obojobo-express/server/config')
 const args = process.argv.slice(2).join(' ')
 const isCreateMode = args.includes('create')
 
 let command
 
 if (!isCreateMode) {
 	const tempDir = fs.mkdtempSync(`${os.tmpdir()}/obojobo-migrations-`)
-
+	const configPath = `${tempDir}/dbconfig.json`
+	console.log(tempDir)
 	try {
 		// useful to understand how args are being inserted into the command
 		command = `${dbMigratePath} ${args} --config ${configPath} --migrations-dir ${tempDir}`
 
+		// use obojobo's config parser to create a json file for db-migrate
+		fs.writeFileSync(configPath, JSON.stringify({development: config.db, production: config.db}, null, '\t'))
```

Comment:
This little change asks obojobo's normal config to process the config files, convert them back into json - and write them in a format that db-migrate will fall use.

Bonus: NODE_ENV will govern the config chosen when running the server and when running the migrations.

Bonus: Obojobo allows ENV variables ending in `_JSON` to be automatically expanded in db.json config.  db-migrate didn't handle those.  Making it easy to forget that you couldn't use json expansion in db.json in some situations.  This fixes that, so there should be no differences!